Road/source/features/data_cleaning.py

```diff
@@ -1,20 +1,20 @@
 # import needed libraries
-import pandas as pd 
-import numpy as np 
+import pandas as pd
+import numpy as np
 import re
-import string 
+import string
 
 # Local imports
 # from source.constants import PUNCT_TO_REMOVE
 import sys
 import os
 target = os.path.abspath(__file__)
-while(target.split("\\")[-1]!="Road"):
+while(target.split("\\")[-1] != "Road"):
     target = os.path.dirname(target)
-sys.path.append(target) 
+sys.path.append(target)
 
 # For arabic diacritics
-# pip install pyarabic 
+# pip install pyarabic
```

Comment:
No need for this comment

Reply:
that's not my code , you can see the data cleaning PR for this comment